assets/panel/debugger.properties

```diff
@@ -648,6 +648,14 @@ watchExpressionsSeparatorLabel2=\u0020→
 # and its real name (if available).
 functionSearchSeparatorLabel=←
 
+# LOCALIZATION NOTE(gotoLineModal.placeholder): The placeholder
+# text displayed when the user searches for specific lines in a file
+gotoLineModal.placeholder=Go to line...
```

Comment:
Use …, not ...

Reply:
It took me a while to figure out what you meant by this, that the first is a unicode character is hard to discern from the text. Putting them in `pre` fences would help distinguish them i.e.
 `...` vs `…`.

Reply:
Good point, I'll keep it in mind.

(in case you're seeing this comment twice, I re-added comment in the right thread).